ddtrace/tracer/tracer_test.go

```diff
@@ -58,6 +59,7 @@ func TestTracerCleanStop(t *testing.T) {
 		return
 	}
 	var wg sync.WaitGroup
+	finished := make(chan struct{})
 	transport := newDummyTransport()
 
 	n := 5000
```

Comment:
Was it not enough to lower `n`? Can you please explain why this change is needed? I understood that there are a lot of statsd clients created now, but why is that a problem? They should be stopping/starting in accordance with these calls. This is a pretty important test IMHO and we should see if we can find a way to avoid changing it.